docs/content/latest/explore/ysql-language-features/pg-extensions.md

```diff
@@ -154,6 +154,28 @@ On CentOS, the correct path is `/usr/pgsql-11/bin/pg_config`.
 
 ## Using PostgreSQL extensions
 
+### file_fdw example
+
+Firstly, install the extension:
+
+```sql
+CREATE EXTENSION file_fdw;
+```
+
+Create a foreign server:
```

Comment:
Is the term "foreign server" correct here? Or should we say "Create a server from a foreign data wrapper"?

Reply:
Maybe `create a foreign server using the foreign data wrapper` ? or is that redundant 
either way, "foreign server" is in line with PG terminology

Reply:
If it's standard terminology, it's all good. Let's leave well enough alone. :)